fulp_modules/features/clothing/halloween/costumes_2021/halloween_2021.dm

```diff
@@ -25,9 +25,122 @@
+	icon_state = "propeller_suit"
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS|FEET
+	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
+	hoodtype = /obj/item/clothing/head/hooded/propeller_head
+
+/obj/item/clothing/head/hooded/propeller_head
+	name = "Propellerhead"
+	desc = "A reflective gold helmet with a makeshift propeller device fastened ontop."
+	icon = 'fulp_modules/features/clothing/halloween/costumes_2021/shovelknight_item.dmi'
+	worn_icon = 'fulp_modules/features/clothing/halloween/costumes_2021/shovelknight_worn.dmi'
+	icon_state = "propeller_head"
+	body_parts_covered = HEAD
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDEHAIR
+
+/obj/item/clothing/gloves/propeller_gloves
+	name = "propeller gloves"
+	desc = "A tight yet comfortable pair of gloves."
+	icon = 'fulp_modules/features/clothing/halloween/costumes_2021/shovelknight_item.dmi'
+	worn_icon = 'fulp_modules/features/clothing/halloween/costumes_2021/shovelknight_worn.dmi'
+	icon_state = "propeller_gloves"
+
+/obj/item/storage/box/halloween/edition_21/spectreknight
+	theme_name = "2021's Spectre Knight"
+
+/obj/item/storage/box/halloween/edition_21/spectreknight/PopulateContents()
+	new /obj/item/clothing/suit/spectre_suit(src)
+	new /obj/item/clothing/head/spectre_head(src)
+
+/obj/item/storage/box/halloween/edition_21/propellerknight
+	theme_name = "2021's Propeller Knight"
+
+/obj/item/storage/box/halloween/edition_21/propellerknight/PopulateContents()
+	new /obj/item/clothing/suit/hooded/propeller_suit(src)
+	new /obj/item/clothing/gloves/propeller_gloves(src)
+
+//HART
+
+/obj/item/clothing/under/hart_body
+	name = "Hart body"
+	desc = "A well worn suit made from various animal furs and bones."
+	icon = 'fulp_modules/features/clothing/halloween/costumes_2021/hart_item.dmi'
+	worn_icon = 'fulp_modules/features/clothing/halloween/costumes_2021/hart_worn.dmi'
+	icon_state = "hart_body"
+	body_parts_covered = GROIN|LEGS|ARMS
+
+/obj/item/clothing/head/hart_head
+	name = "Hart hood"
+	desc = "A hood made from tattered black cloth."
+	icon = 'fulp_modules/features/clothing/halloween/costumes_2021/hart_item.dmi'
+	worn_icon = 'fulp_modules/features/clothing/halloween/costumes_2021/hart_worn.dmi'
+	icon_state = "hart_head"
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDEHAIR
+
+/obj/item/clothing/mask/lovely_mask
+	name = "Lovely Mask"
+	desc = "A porcelain disk with a crudely drawn on heart."
+	icon = 'fulp_modules/features/clothing/halloween/costumes_2021/hart_item.dmi'
+	worn_icon = 'fulp_modules/features/clothing/halloween/costumes_2021/hart_worn.dmi'
+	icon_state = "lovely_mask"
+	flags_inv = HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR
+
+/obj/item/storage/box/halloween/edition_21/hartcostume
+	theme_name = "2021 - Hart's lovelies"
+
+/obj/item/storage/box/halloween/edition_21/hartcostume/PopulateContents()
+	new /obj/item/clothing/under/hart_body(src)
+	new /obj/item/clothing/head/hart_head(src)
+	new /obj/item/clothing/mask/lovely_mask(src)
```

Comment:
why 3 masks

Reply:
because its hart's lovelies not harts lovely